tools/topology/topology2/include/bench/bench_comp_generate.sh

```diff
@@ -71,32 +71,48 @@ EOF_ROUTE
 
 generate_playback_controls ()
 {
-    fn=${comp}_control_bytes_playback.conf
+    fn=${comp}_controls_playback.conf
     echo Creating file "$fn"
     cat > "$fn" <<EOF_PLAYBACK_BYTES
 			# Created initially with script "${FULL_CMD[@]}"
 			# may need edits to modify controls
-			Object.Control.bytes."1" {
-				name '\$ANALOG_PLAYBACK_PCM ${comp^^} bytes'
-				IncludeByKey.BENCH_${comp^^}_PARAMS {
-					"default" "include/components/${comp}/default.conf"
-				}
+			Object.Control {
+				#bytes."1" {
+				#	name '\$ANALOG_PLAYBACK_PCM ${comp^^} bytes'
+				#	IncludeByKey.BENCH_${comp^^}_PARAMS {
+				#		"default" "include/components/${comp}/default.conf"
+				#	}
+				#}
+				#mixer."1" {
+				#	name '\$ANALOG_PLAYBACK_PCM ${comp^^} switch or volume'
+				#}
+				#enum."1" {
+				#	name '\$ANALOG_PLAYBACK_PCM ${comp^^} enum'
+				#}
 			}
```

Comment:
Meant to be commented out ?

Reply:
Yes, the idea is to un-comment and adapt name of the controls those are supported by the component.

Reply:
Some script could find those out from component class .conf or from module source code but in this step it's manual work.

Reply:
ok, this assumption should be commented above the comment block so that everyone knows the usage and future path.

Reply:
Yep, I added the comments.